scripts/extensions/pool_update.precheck

```diff
@@ -49,16 +50,25 @@ def parse_control_package(session, yum_url):
     return items[0].getAttribute('control')
 
 
-def execute_precheck(session, control_package, yum_conf_file, nogpgcheck):
-    FNULL = open(os.devnull, 'w')
-    retcode = subprocess.call('yum -c ' + yum_conf_file + ' install -y ' +
-                              nogpgcheck + control_package, shell=True, stdout=FNULL, stderr=FNULL)
-    if retcode != 0:
+def execute_precheck(session, control_package, yum_conf_file):
+    cmd = ['yum', 'install', '-y', '--noplugins', '-c', yum_conf_file, control_package]
+    p = subprocess.Popen(cmd, shell=False, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, close_fds=True)
+    p.wait()
+    output = p.stdout.read()
+    xcp.logger.info('pool_update.precheck %r returncode=%r output=%r', cmd, p.returncode, output)
```

Comment:
I'm not sure but if this is for updating RPMs that we control the output of `yum` should not reach a size where we need to worry about this. I would be surprised if this is even above 10k.